.gitignore

```diff
@@ -1,3 +1,5 @@
 _build
+_opam
+.ocamlformat
```

Comment:
Added for developer comfort 😄 
Let me know if you'd rather not and I'll remove it.